test/helpers.coffee

```diff
@@ -123,3 +123,10 @@ test "the `baseFileName` helper returns the file name to write to", ->
     name = baseFileName sourceFileName, yes
     filename = name + ext
     eq filename, expectedFileName
+
+test "the `updateSyntaxError` helper allows `throw null`", ->
```

Comment:
Without the try/catch the previous code would throw a `TypeError` that would not be caught in a tidy fashion from the test. If this is not considered an issue I can change it. I think a better place for testing this fix is `test/exception_handling.coffee` with a simple `throws (->throw null), null`. Should I proceed thus?